DDR4 OC Guide.md

```diff
@@ -476,7 +476,7 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
    | tWTRS tWTRL | 4 12 | 4 10 | 4 8 |
    | tRTP | 12 | 10 | 8 |
    | tCWL<sup>1</sup> | tCL | tCL - 1 | tCL - 2 |
-   * On Intel, tWTRS/L should be left on auto and controlled with tWRRD_dg/sg respectively. Dropping tWRRD_dg by 1 will drop tWTRS by 1. Likewise with tWRRD_sg. Once they're as low as you can go, manually set tWTRS/L.
+   * Like tWR, on Intel tWTRS/L are not real timings and are replaced by tWRRD_dg and tWRRD_sg respectivly. Due to this, dropping tWRRD_dg will lower tWTRS by 1. Likewise with tWRRD_sg. In tWTRS/L readings it must be noted that the timings are read two lower then they physically, meaning a reading of 4/8 tWTRS/L in asrock timing configurator is actually 6/10. Conversion between tWTRS/L and tWRRD_sg/dg readings is tWTR = tWRRD - tCWL - 6. However this is two lower then actual, tWTR = tWWRD - tCWL - 4 is the actual effective tWTR.
```

Comment:
> it must be noted that the timings are read two lower then they physically, meaning a reading of 4/8 tWTRS/L in asrock timing configurator is actually 6/10.

Source?

Reply:
![image](https://user-images.githubusercontent.com/77159913/121153633-91601500-c889-11eb-9866-aba4253b5f9c.png)
tWTR is the gap between the end of the write burst and the read command.
tWRRD is the write to read command.
The length in clocks between the write command and end of the write burst is write latency (equal to tCWL without AL) plus the length of the burst, which is 4 clocks as a standard burst and 2 if burst chop is used.
So tWTR = tWRRD - tCWL - 4 for BL and tWTR = tWRRD - tCWL - 2 for BC.
BC isn't used on these platforms afaik, though that could be wrong.
And in software as shown here
![image](https://user-images.githubusercontent.com/77159913/121154084-fc115080-c889-11eb-8752-16c851e7a86c.png)
tWTR = tWRRD - tCWL - 6, which doesn't make sense at all. So it is being read 2 lower with BL or 4 lower if BC is used.

Reply:
![image](https://user-images.githubusercontent.com/77159913/121154370-472b6380-c88a-11eb-93bf-735ec1a708e9.png)